From 5be1d795ec858f47aadd9ffcc9a2360891023050 Mon Sep 17 00:00:00 2001
From: Eisuke Kawashima <e-kwsm@users.noreply.github.com>
Date: Wed, 16 Oct 2019 07:36:59 +0900
Subject: [PATCH] Install future only if python2

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index bddfd29..d575fb9 100755
--- a/setup.py
+++ b/setup.py
@@ -32,7 +32,7 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'future>=0.14.0',
+        'future>=0.14.0; python_version<"3"',
         'futures; python_version<"3.2"',
     ],
